penlight-scm-3.rockspec

```diff
@@ -12,9 +12,10 @@ description = {
   license = "MIT/X11",
   maintainer = "thijs@thijsschreijer.nl",
   detailed = [[
-Penlight is a set of pure Lua libraries for making it easier to work with common tasks like
-iterating over directories, reading configuration files and the like. Provides functional operations
-on tables and sequences.
+Penlight is a set of pure Lua libraries focusing on input data handling
+(such as reading configuration files), functional programming
+(such as map, reduce, placeholder expressions,etc), and OS path management.
+Much of the functionality is inspired by the Python standard libraries.
```

Comment:
this can be indented to properly line up. LuaRocks will remove the trailing whitespace automatically.

Reply:
You mean leading whitespace? If so that's good to know, I thought I saw this broken on some projects, maybe that is something that has been fixed. I'd love not to have to do flush-left alignment like this.

Reply:
I'm moving the commit that touched the rockspec to another PR anyway.